src/applications/discharge-wizard/components/CarefulConsiderationStatement.jsx

```diff
@@ -47,7 +47,8 @@ const reasonStatement = formValues => {
             punishment for those events.
           </p>
         );
-      } else if (dischargeType === '1') {
+      }
+      if (dischargeType === '1') {
```

Comment:
The linter keeps changing this. But it's ok not to be an `else if` because the condition above it and this one cannot be true at the same time.